code/datums/supplypacks.dm

```diff
@@ -2019,6 +2029,18 @@ AMMO
 	containername = "ammo crate"
 	group = "Ammo"
 
+/datum/supply_packs/ammo_minigun
+	name = "Vindicator Minigun Ammo Drums (x3)"
+	contains = list(
+					/obj/item/ammo_magazine/minigun,
+					/obj/item/ammo_magazine/minigun,
+					/obj/item/ammo_magazine/minigun)
+	cost = RO_PRICE_CHEAP
```

Comment:
Should be `RO_PRICE_KINDA_PRICY`, considering it's one thousand and half ammunitions.